nixos/modules/services/monitoring/todesk.nix

```diff
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  cfg = config.services.todesk;
+
+in
+
+{
+
+  ###### interface
+  options = {
+
+    services.todesk.enable = lib.mkEnableOption "ToDesk daemon";
```

Comment:
Are there no settings?

Reply:
I meant a `package` option to overide `pkgs.todesk`. See e.g. https://search.nixos.org/options?channel=unstable&show=services.samba.package&from=0&size=50&sort=relevance&type=packages&query=samba